README.md

```diff
@@ -9,23 +9,22 @@ for those systems, and specified telemetry or data from various tools (`sar`,
 
 The second sub-system is the `pbench-server`, which is responsible for
 archiving result tar balls, indexing them, and unpacking them for display.
+It provides a RESTful API which can be used by client applications, such as
+the Pbench Dashboard, which provides a web-based interface to the Pbench
+Server.
```

Comment:
Although the "dashboard" stuff probably doesn't belong here, I'd have preferred to keep the "RESTful API" reference. Also, technically, the "unpacking them for display" is a bit outdated, and maybe a more general "managing access to components/elements/files/inventory within them?"???

Reply:
Let me know if the result needs...more.